doc/dev/tests.md

```diff
@@ -0,0 +1,378 @@
+```cmd
+(env)azure-sdk-for-python\sdk\my_directory\my_library> python
+
+>>> import azure.my_library
+>>> print(azure.my_library.__version__)
+0.0.1
+```
+
+### Open code in IDE
+
+Open the directory for your library in your preferred editor, for example VSCode.
+```cmd
+(env)azure-sdk-for-python\sdk\my_directory\my_library> code .
+```
+
+## Integrate with the pytest Test Framework
+As a quick background, the Azure SDK uses the [pytest](https://docs.pytest.org/en/latest/) test runner to support creating unit and functional tests for Track 2 Azure libraries. To intall `pytest` run `pip install pytest` from your virtual environment, you can confirm the installation was successful by running `pytest -V`. The commands will run all files of the form `test_*.py` or `*_test.py` in the provided directory and its subdirectories, for more information check out the [docs](https://docs.pytest.org/en/stable/getting-started.html#run-multiple-tests).
+
+With the pytest test suite you can provide directories or specific tests to run rather than running the entire test suite:
+```cmd
+azure-sdk-for-python\sdk\my_directory\my_library> pytest
+azure-sdk-for-python\sdk\my_directory\my_library> pytest <test_file.py>
+```
+
+In addition you can provide keywords to run specific tests within the suite or within a specific file
+```cmd
+azure-sdk-for-python\sdk\my_directory\my_library> pytest -k <keyword>
+azure-sdk-for-python\sdk\my_directory\my_library> pytest <test_file.py> -k <keyword>
+```
+
+If you have print statements in your tests for debugging you can add the `-s` flag to send those print statements to standard output:
+```cmd
+azure-sdk-for-python\sdk\my_directory\my_library> pytest sdk/storage/azure-mgmt-storage/ -s
+```
+
+## Tox
+The Python SDK uses the [tox project](https://tox.readthedocs.io/en/latest/) to automate releases, run tests, run linters, and build our documentation. The `tox.ini` file is located at `azure-sdk-for-python/eng/tox/tox.ini` for reference. You do not need to make any changes to the tox file for tox to work with the `app-config` project. Tox will create a directory (`.tox`) in the head of your branch. The first time you run tox commands it may take several moments, but subsequent runs will be quicker. To install tox run the following command from within your virtual environment.
+`(env) > pip install tox tox-monorepo`.
+
+To run a tox command from your directory use the following commands:
+```cmd
+azure-sdk-for-python\sdk\my_directory\my_library> tox -c ../../../eng/tox/tox.ini -e sphinx
+azure-sdk-for-python\sdk\my_directory\my_library> tox -c ../../../eng/tox/tox.ini -e lint
+azure-sdk-for-python\sdk\my_directory\my_library> tox -c ../../../eng/tox/tox.ini -e mypy
+azure-sdk-for-python\sdk\my_directory\my_library> tox -c ../../../eng/tox/tox.ini -e whl
+azure-sdk-for-python\sdk\my_directory\my_library> tox -c ../../../eng/tox/tox.ini -e sdist
```

Comment:
please add 😄 

azure-sdk-for-python\sdk\my_directory\my_library> tox -c ../../../eng/tox/tox.ini -e samples